mango-core/src/main/scala/org/bdgenomics/mango/layout/MismatchLayout.scala

```diff
@@ -259,8 +258,8 @@ object PointMisMatch {
   }
 }
 
-// tracked MisMatch Json Object
-case class MisMatchJson(op: String, refCurr: Long, length: Long, sequence: String, refBase: String, track: Long)
+// MisMatch Json Object
+case class MisMatchJson(op: String, refCurr: Long, length: Long, sequence: String, refBase: String)
```

Comment:
We should keep mismatch, not mismatchjson. If you eliminated mismatchjson, just change the name here.